scripts/release-stable.ts

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env node
+
+import fs from 'fs-extra';
+import prompts from 'prompts';
+import sh from 'shelljs';
+
+import { resolveDir, changeAllPkgJSON, restorePkgJson } from './utils/changePackages';
+
+(async () => {
+  const mainPkgJSON = await fs.readJSON(resolveDir('./package.json'));
+
+  /**
+   * Change all package.json with the version from main package.json
+   * Run lerna to generate changelogs and bump version based on conventional commits
+   */
+  await changeAllPkgJSON(mainPkgJSON.version);
+  const args = `--conventional-commits --no-push --no-git-tag-version --no-commit-hooks --no-ignore-changes --yes`;
```

Comment:
This sounds a bit problematic because if anything breaks during the running of this, we will need to checkout or do another manual reversion of the files.

The probability looks low, and also if running on CI it only will need to be re-runned.

Reply:
Cool, did some kinda a `trycatch` making a `git checkout .` if some error occurs. But, because of that, the git commands are just executing after all release/changelogs/version happens.